src/utils/firebase/client.js

```diff
@@ -0,0 +1,37 @@
+import firebase from 'firebase/app';
+
+import 'firebase/auth';
+import 'firebase/firestore';
+
+export default new Proxy(
```

Comment:
Whoa why is this a Proxy?

Reply:
So we can call a function everytime a method is called in an object without blocking its execution